wheat_filtration/mallet.py

```diff
@@ -1,68 +1,80 @@
 from collections import OrderedDict
 import os
-import shlex
 import subprocess
+import warnings
 
 import gensim.corpora as corpora
 from gensim.models.wrappers import LdaMallet
 import numpy as np
 from nltk.corpus import stopwords
+from scipy.sparse import coo_matrix
 
 import munge
 
+# not sure what to do with this... vs code keeps moving it
+# do i even need to keep it in? can we just assume theyve done this?
+# import nltk
+# nltk.download('stopwords')
 
-MALLET_PATH = None  # "/Users/fauma/Mallet-master/bin/mallet"  #
 
+MALLET_PATH = "/Users/fauma/Mallet-master/bin/mallet"  # None  #
 
-def _call_command_line(string):
-    """Executes string as a command line prompt."""
-    return subprocess.call(shlex.split(string))
+
+def _call_command_line(string, **kwargs):
+    """Executes string as a command line prompt. stdout and stderr are keyword args."""
+    return subprocess.run(string, **kwargs)
 
 
 def _make_doc_dictionary(path_to_mallet, mallet_instance_filepath):
```

Comment:
These helpers might actually belong in the class. You can define them with `@classmethod` decorators if you want and add the first argument `cls` to the methods so you can just call `TopicModel._function(args)` without needing to instantiate an object.